docs/user-guide/configure.md

```diff
@@ -559,6 +560,18 @@ For example:
 
 [More info](options.md#reportneedlessdisables).
 
+### `reportUnscopedDisables`
+
+Report configuration comments that are not scoped to at least one rule. A [`report*`](#report) property.
+
+For example:
+
+```json
+{
+  "reportUnscopedDisables": true
+}
+```
+
```

Comment:
[suggestion] Same here for "More info" link:

```suggestion

[More info](options.md#reportunscopeddisables).

```

Reply:
see https://github.com/stylelint/stylelint/pull/8024#discussion_r1796435578